packs/trello/sensors/list_actions_sensor.yaml

```diff
@@ -9,14 +9,38 @@ trigger_types:
     payload_schema:
       type: object
       properties:
+        id:
+          description: "Unique action ID"
+          type: string
         data:
+          description: "Structurized data returned for action, depends on action type"
           type: object
+        date:
+          description: "When action occurred"
+          type: string
+          format: date-time
         idMemberCreator:
+          description: "User ID who initiated the action"
+          type: string
+        type:
+          description: "Action type (ex: createCard)"
           type: string
         memberCreator:
+          description: "Additional info about user who initiated action"
           type: object
-        date:
-          type: string
-          #format: 2015-09-02T18:53:30.900Z
-        id:
-          type: string
+          properties:
```

Comment:
Nested properties in Sensor `payload_schema` first try,
cc https://github.com/StackStorm/st2web/issues/219